app/scripts/directives/notifications/notificationDrawerWrapper.js

```diff
@@ -184,7 +185,7 @@
           // using uid to match API events and have one filed to pass
           // to EventsService for read/cleared, etc
           trackByID: notification.trackByID,
-          uid: id,
+          uid: _.includes(id, project) ? id : project + "/" + id,
```

Comment:
I think we should always prepend project rather than trying to check if it's in `id`. There's no harm and it makes things simpler.

Reply:
Ok, fixed -thanks

Reply:
Agree on keeping it simpler. If the `id` is being created on `line 180`, can we keep all the `id` manipulation logic there?  Manipulating it in a couple spots might open doors to bugs...